source/command-line-options.txt

```diff
@@ -33,18 +35,75 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line:
+
+.. option:: --autoUpdates
+  
+   Enable :ref:`automatic updates <enable-automatic-updates>`.
+  
+.. option:: --enableFeedbackPanel
+  
+   Enable the feedback panel.
+   
+.. option:: --enableMaps
+ 
+   Enable geographic visualizations.
+
+.. option:: --exportConnections
+    
+   :ref:`Export <compass-export-connections-cli>` favorite connections.
+
+.. option:: --file <file>
+  
+   Specify a file containing a :ref:`list of connections 
+   <compass-export-connections-cli>`. These connections are 
+   automatically available once |compass| starts.
+
+.. option:: --help
+   
+   Show |compass| options.
+
+.. option:: --importConnections
+
+   :ref:`Import <compass-import-connections-cli>` favorite connections.
+
+.. option:: --lgDarkmode
+
+   Use leafygreen dark mode instead of darkreader.
+
+.. option:: --networkTraffic
+
+   Configure |compass| to only allow outgoing :ref:`network operations 
+   <compass-configure-network-traffic>` to connect to the database.
+
+.. option:: --passphrase
+ 
+   Specify a passphrase to encrypt the :ref:`exported favorite 
+   connections <compass-export-connections-cli-encrypted>` output file.
+
+   Specify a passphrase to decrypt the :ref:`imported favorite
+   connections <compass-import-connections-cli-encrypted>` file. 
+
+.. option:: --protectConnectionStrings
+
+   Protect connection string secrets.
```

Comment:
Do you want to include something like the long-form explanation that we include for most options in the settings panel?

Reply:
I expanded this description with info from the **Settings** panel. Thanks!